src/App.vue

```diff
@@ -66,7 +66,7 @@
               </h2>
               <Service
                 v-for="item in group.items"
-                :key="item.name"
+                :key="item.url"
```

Comment:
Just to be 100% covered, I would go for something like that (untested, but should work): 
```js
v-for="(item, index) in group.items"
:key="index"
```
That way even duplicate urls won't be an issue

Reply:
You're right. That's similar to how AngularJS did it.
Fixed it & tested it.